pom.xml

```diff
@@ -5,67 +5,69 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>elasticsearch.concatenate</groupId>
 	<artifactId>elasticsearch-concatenate</artifactId>
-	<version>1.1.0</version>
+	<version>5.4.2</version>
```

Comment:
Is that the plugin's version? Is it typical for community / 3rdparty plugins to list their version as exactly the ES version?